### PR TITLE
XW-3349 | Preserve scroll position when closing lightbox.

### PR DESCRIPTION
**Pull request by JamesHawking**
## Links

https://wikia-inc.atlassian.net/browse/XW-3349

## Description

Preserve scroll position when closing lightbox.
Upgrade ember-router-scroll to 0.2.0.

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: app/controllers/wiki-page.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
 import Ember from 'ember';
 
 export default Ember.Controller.extend({
-	application: Ember.inject.controller()
+	application: Ember.inject.controller(),
+	actions: {
+		/**
+		 *
+		 * @param lightboxType
```
Comment (hakubo) [inline app/controllers/wiki-page.js]: missing types

File: app/routes/wiki-page.js [hunk 2]
```diff
@@ -58,6 +58,14 @@ export default Route.extend(
 		},
 
 		/**
+		 *
+		 * @param controller
```
Comment (hakubo) [inline app/routes/wiki-page.js]: missing type

**Review by hakubo — CHANGES_REQUESTED** [review]

**Review by hakubo — APPROVED** [review]

**Review by rybmat — APPROVED** [review]

File: app/routes/wiki-page.js [hunk 3]
```diff
@@ -58,6 +58,14 @@ export default Route.extend(
 		},
 
 		/**
+		 *
+		 * @param {Ember.Controller} controller
+		 */
```
Comment (rogatty) [inline app/routes/wiki-page.js]: `@returns {void}`

File: app/routes/wiki-page.js [hunk 4]
```diff
@@ -274,7 +282,24 @@ export default Route.extend(
...
+			 */
+			openLightbox(lightboxType, lightboxModel, closeButtonDelay) {
+				this.get('controller').send('openLightbox', lightboxType, lightboxModel, closeButtonDelay);
+			},
+
+			/**
+			 * @returns {void}
+			 */
+			closeLightbox() {
+				this.get('controller').send('closeLightbox');
```
Comment (rogatty) [inline app/routes/wiki-page.js]: Is this needed? There is exactly the same thing in `routes/application.js`

**Comment by ludwikkazmierczak** [comment]
Tested and Accepted

**Pull request merged**